CMakeLists.txt

```diff
@@ -29,14 +29,20 @@ include( "demos/logging-stack/logging.cmake" )
 
 # Configure options to always show in CMake GUI.
 option( BUILD_TESTS
-        "Set this to ON to build both demo and test executables. When OFF, only demo executables are built."
+        "Set this to ON to build test executables."
         OFF )
+option( BUILD_DEMOS
```

Comment:
The `build-check-demos` job will have to be updated to pass this flag so that the demo targets are available in the job run